docs/developers-guide/dev-branch-docker.md

```diff
@@ -26,7 +26,7 @@ Once Docker is installed, you’re ready to go.
 2. Copy and paste this command, switching out `<branch-name>` for the name of the branch you’d like to test:
 
 ```bash
-docker run -d -p 3000:3000 metabase/metabase-dev:<branch-name>
+docker run --platform linux/amd64 -d -p 3000:3000 --name metabase-dev metabase/metabase-dev:<branch-name>
```

Comment:
I tried running this command with `release-x.42.x` as the branch, but I got a manifest error.